SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -246,21 +246,35 @@ void USpatialReceiver::ReceiveWorkerDisconnectResponse(const Worker_CommandRespo
 
 void USpatialReceiver::ReceiveClaimPartitionResponse(const Worker_CommandResponseOp& Op)
 {
-	const Worker_PartitionId PartitionId = PendingPartitionAssignments.FindAndRemoveChecked(Op.request_id);
+	if (Op.request_id < 0)
+	{
+		// Invalid request id that will not be in PendingPartitionAssignments
+		return;
+	}
+
+	const Worker_PartitionId* PartitionId = PendingPartitionAssignments.Find(Op.request_id);
+	if (PartitionId == nullptr)
+	{
+		UE_LOG(LogSpatialVirtualWorkerTranslationManager, Error, TEXT("Could not find request id %d in PendingPartitionAssignments"),
```

Comment:
I don't think this is an error. For system commands like this I don't think we can easily make assumptions about who might have sent a command request.